infracost-usage-example.yml

```diff
@@ -158,6 +158,9 @@ resource_usage:
     monthly_log_lines: 5000000          # Monthly number of real-time log lines.
     custom_ssl_certificates: 3          # Number of dedicated IP custom SSL certificates.
 
+  aws_autoscaling_group.my_asg:
+    instances_count: 15 # Number of instances in the autoscaling group.
```

Comment:
We use `instances` elsewhere and in the contributing docs (https://github.com/infracost/infracost/blob/master/CONTRIBUTING.md#usage-file-notes), so I think we should keep this consistent.

Reply:
Unfortunately, I didn't exactly get your suggestion. I believe `count` itself does not make much sense in this use case.

Reply:
`instances_count` -> `instances`

We use `instances` elsewhere as well.

Reply:
Ahh sorry, my bad. Will fix it.